src/main/java/six/six/gateway/twilio/TwilioSmsService.java

```diff
@@ -0,0 +1,19 @@
+package six.six.gateway.twilio;
+
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+import org.jboss.logging.Logger;
+import six.six.gateway.SMSService;
+
+public class TwilioSmsService implements SMSService {
+  private static final Logger logger = Logger.getLogger(TwilioSmsService.class);
+
+  @Override
+  public boolean send(String phoneNumber, String message, String login, String pw) {
+    Twilio.init(login, pw);
+
+    Message sms = Message.creator(new PhoneNumber(phoneNumber), new PhoneNumber("+15815015124"), message).create();
```

Comment:
https://www.twilio.com/docs/messaging/services?code-sample=code-send-a-message-with-a-messaging-service&code-language=Java&code-sdk-version=6.x

I think we already have a service id, as we have 'twilio_service_sid' in our vault.  So probably best to add this as a configuration parameter somewhere.

Reply:
It's either the phoneNumber we're sending from or the serviceId. ServiceId is used a lot (probably mandatory) in the verification API, however, in the messaging (SMS) API it's hardly used nor documented.